zkchannels-contract/zkchannel_smartpy_script.py

```diff
@@ -161,46 +184,52 @@ def custClose(self, custBal, merchBal, revLock, s1, s2):
         # Note that all addresses must be implicit accounts (tz1), not smart contracts
         sp.if merchBal != sp.tez(0):
             sp.send(self.data.merchAddr, merchBal)
-        self.data.merchBal = sp.tez(0)
         self.data.status = CUST_CLOSE
  
     # merchDispute can be called if the merchant has the secret corresponding
     # to the latest custClose state. If the secret is valid, the merchant will
     # receive the customer's balance too.
```

Comment:
"revocation secret corresponding to the revocation lock in the customer's custClose entrypoint call". I don't know what "latest custClose state" means here.

Reply:
updated to
```
    # merchDispute can be called by the merchant if they have the revocation secret corresponding 
    # to the revocation lock in the customer's custClose entrypoint call. If the secret is valid, 
    # the merchant will receive the customer's balance.
````